bitnami/concourse/templates/NOTES.txt

```diff
@@ -0,0 +1,29 @@
+
+* Concourse can be accessed:
+
+  * Within your cluster, at the following DNS name at port {{ .Values.web.bindPort }}:
+
+    {{ template "concourse.web.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+
+
+  * From outside the cluster, run these commands in the same shell:
+    {{- if contains "NodePort" .Values.web.service.api.type }}
+
+    export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "concourse.web.fullname" . }})
+    export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+    echo http://$NODE_IP:$NODE_PORT
+    {{- else if contains "LoadBalancer" .Values.web.service.api.type }}
+
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get svc -w {{ template "concourse.web.fullname" . }}'
+
+    echo http://$SERVICE_IP:{{ .Values.web.bindPort }}
+    {{- else if contains "ClusterIP"  .Values.web.service.api.type }}
+
+    export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "concourse.web.fullname" . }}" -o jsonpath="{.items[0].metadata.name}")
+    echo "Visit http://127.0.0.1:8080 to use Concourse"
+    kubectl port-forward --namespace {{ .Release.Namespace }} $POD_NAME 8080:{{ .Values.web.bindPort }}
+    {{- end }}
+
+* If this is your first time using Concourse, follow the examples at https://concourse-ci.org/examples.html
+
```

Comment:
Missing here a call to validateValues and the rolling tags check. Check other charts